app/views/govuk_publishing_components/components/_show_password.html.erb

```diff
@@ -0,0 +1,35 @@
+<%
+  disable_form_submit_check ||= false
+  label ||= nil
+  value ||= nil
+  id ||= nil
+  name ||= nil
+  describedby ||= nil
+  hint ||= nil
+  error_message ||= nil
+  error_items ||= nil
+%>
+<% if label %>
```

Comment:
It looks like this would show nothing if you don't pass in a label, is that intentional? Might be that we want to raise an error in that case.

Reply:
Have updated to error if no label provided.